{{cookiecutter.project_shortname}}/scripts/bootstrap

```diff
@@ -27,3 +27,4 @@ pipenv run invenio collect -v
 pipenv run invenio webpack buildall
 
 cp -f $script_path/../static/images/logo.svg $(pipenv run invenio shell --no-term-title -c "print(app.instance_path)")/static/images/logo.svg
+cp -f $script_path/../invenio.cfg $(pipenv run invenio shell --no-term-title -c "print(app.instance_path)")/invenio.cfg
```

Comment:
normal: The problem with copying is that it muddies which invenio.cfg is taken into account. Since we know the `app.instance_path)")/invenio.cfg` one to be picked up, I would link it or move it.

Reply:
symlink it is :)